pulsar-io/hdfs3/src/test/java/org/apache/pulsar/io/hdfs3/sink/seq/HdfsTextSinkTests.java

```diff
@@ -21,85 +21,85 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.testng.Assert.assertNotNull;
-
 import org.apache.pulsar.io.hdfs3.sink.AbstractHdfsSinkTest;
-import org.apache.pulsar.io.hdfs3.sink.seq.HdfsTextSink;
 import org.testng.annotations.Test;
 
 public class HdfsTextSinkTests extends AbstractHdfsSinkTest<String, String> {
-    
+
     @Override
     protected void createSink() {
         sink = new HdfsTextSink();
     }
 
-    @Test(enabled = false)
+    @Test
     public final void write100Test() throws Exception {
         map.put("filenamePrefix", "write100TestText-seq");
         map.put("fileExtension", ".seq");
         map.put("syncInterval", 1000);
         sink.open(map, mockSinkContext);
-        
+
         assertNotNull(sink);
         assertNotNull(mockRecord);
         send(100);
-        
+
         Thread.sleep(2000);
         verify(mockRecord, times(100)).ack();
         sink.close();
     }
-    
-    @Test(enabled = false)
+
+    @Test
     public final void write5000Test() throws Exception {
         map.put("filenamePrefix", "write5000TestText-seq");
         map.put("fileExtension", ".seq");
         map.put("syncInterval", 1000);
         sink.open(map, mockSinkContext);
-        
+
         assertNotNull(sink);
         assertNotNull(mockRecord);
         send(5000);
-        
+
         Thread.sleep(2000);
         verify(mockRecord, times(5000)).ack();
         sink.close();
     }
-    
-    @Test(enabled = false)
+
+    @Test
     public final void tenSecondTest() throws Exception {
         map.put("filenamePrefix", "tenSecondTestText-seq");
         map.put("fileExtension", ".seq");
         map.put("syncInterval", 1000);
         sink.open(map, mockSinkContext);
-        
+
         assertNotNull(mockRecord);
-        
-        runFor(10); 
+
+        runFor(10);
         sink.close();
     }
-    
-    @Test(enabled = false)
+
+    @Test
     public final void bzip2CompressionTest() throws Exception {
         map.put("filenamePrefix", "bzip2CompressionTestText-seq");
         map.put("compression", "BZIP2");
         map.remove("fileExtension");
         sink.open(map, mockSinkContext);
-        
+
         assertNotNull(mockRecord);
-        
+
         send(5000);
+        sink.close();
         verify(mockRecord, times(5000)).ack();
     }
-    
-    @Test(enabled = false)
+
+    @Test
     public final void deflateCompressionTest() throws Exception {
         map.put("filenamePrefix", "deflateCompressionTestText-seq");
         map.put("compression", "DEFLATE");
         map.remove("fileExtension");
         sink.open(map, mockSinkContext);
-        
+
         assertNotNull(mockRecord);
         send(5000);
+        sink.close();
```

Comment:
This pattern is the only unstable cause that if we don't close (internally stop and join the thread) the sink, it can be inflight ack that is not counted in the `times` checker.